node/chainSimulator/chainSimulator_test.go

```diff
@@ -77,6 +78,12 @@ func TestChainSimulator_GenerateBlocksShouldWork(t *testing.T) {
 		InitialRound:                200000000,
 		InitialEpoch:                100,
 		InitialNonce:                100,
+		AlterConfigsFunction: func(cfg *config.Configs) {
+			// we need to enable this as this test skips a lot of epoch activations events, and it will fail otherwise
+			// because the owner of a BLS key coming from genesis is not set
+			// (the owner is not set at genesis anymore because we do not enable the staking v2 in that phase)
```

Comment:
This test should pass regardless of this. Something is wrong if this fails without this enable epoch.